ERROR:pymodbus.logging:[Errno 11] Could not exclusively lock port /dev/ttyACM0: [Errno 11] Resource temporarily unavailable
ERROR:pymodbus.logging:[Errno 11] Could not exclusively lock port /dev/ttyACM0: [Errno 11] Resource temporarily unavailable
Traceback (most recent call last):
  File "/Energiebox/Tracer/readall.py", line 20, in <module>
    response = client.execute(request)
  File "/Energiebox/Tracer/pymodbus/client/base.py", line 395, in execute
    raise ConnectionException(f"Failed to connect[{self!s}]")
pymodbus.exceptions.ConnectionException: Modbus Error: [Connection] Failed to connect[ModbusSerialClient /dev/ttyACM0:0]
Fehler: Konnte readall.py nicht ausf端hren! Port und Baudrate pr端fen!
Fehler: Konnte readall.py nicht ausf端hren! Port und Baudrate pr端fen!
